developer/tutorial/creating-a-payment-provider-plugin.md

```diff
@@ -2,7 +2,7 @@
 
 ## What is a "Payment Provider"
 
-Payment Providers are companies that provide a gateway between ecommerce
+Payment Providers are companies that provide a gateway between e-commerce
```

Comment:
Our official style is "ecommerce" actually. (I am working on getting our internal word style guide out!)

Reply:
good to know!